### PR TITLE
Adjust statistical analysis for the updated pipeline and exploration

### LINKED ISSUE #50: Remove T2w processing in processing pipeline
## Context
With the segmentation results from the first 350 subjects, we decided to only use T1w images for the project:
* Subjects to correct segmentation
    * T1w: 15/350
    * T2w: 50/350
* Subjects to exclude→ 47 / 700 from QC report
    * T1w : 6/350
    * T2w : 41/350

This implicates that the processing pipeline needs to be adjusted to only generate sc segmentations and vertebral lableing for T1w images (mainly in process_data.sh) .
## Tasks
The removal of T2w images implicates the following changes:
- [x] Remove T2w processing from `process_data.sh`
- [x] Adapt `compute_stats.py` to remove T2w from the statistical models
- [x] Adjust `README.md` mainly [Data collection and organization](https://github.com/sct-pipeline/ukbiobank-spinalcord-csa#data-collection-and-organization)

### PR DESCRIPTION
**Pull request by sandrinebedard**
## Description
This PR includes various modifications of `get_subject_info.py` and `compute_stats.py` to address #50 and to add new parameters and results for exploration of the CSA results of the first 1000 subjects.

Main changes include:
* Remove T2w CSA analysis
* Add new parameters
* Adapt exclusion of subject with current exclude.yml format
* Gender- CSA analysis (mean CSA per gender +  T-Test +plots)
* Age -  CSA analysis (linear regression + quadratic regression)
* Add scatter plots of each parameter and CSA:
example : 

## Linked issue
Closes #50

### CODE REVIEW

**Comment by jcohenadad** [comment]
> Gender- CSA analysis (mean CSA per gender + T-Test +plots)

I would call it Sex (which is the biological male/female definition that we are referring to here, while the term "gender" refers to the identity and social construct, see [here](https://cihr-irsc.gc.ca/e/50836.html))

File: pipeline_ukbiobank/cli/compute_stats.py [hunk 1]
```diff
@@ -143,25 +142,25 @@ def compute_predictors_statistic(df):
         stats[predictor]['med'] = np.median(df[predictor])
         stats[predictor]['mean'] = np.mean(df[predictor])
     # Computes male vs female ratio
-    stats['Sex'] = {}
-    stats['Sex']['%_M'] = 100*(np.count_nonzero(df['Sex']) / len(df['Sex']))
-    stats['Sex']['%_F'] = 100 - stats['Sex']['%_M']
+    stats['Gender'] = {}
```
Comment (jcohenadad) [inline pipeline_ukbiobank/cli/compute_stats.py:145]: Change back to 'Sex' as per my comment https://github.com/sct-pipeline/ukbiobank-spinalcord-csa/pull/74#issuecomment-850544868.  I'm curious about what made you change it from Sex to Gender?

Comment (sandrinebedard) [reply pipeline_ukbiobank/cli/compute_stats.py:145]: I honestly don't remember 😬 I don't think it was that well thought out, I juste checked in UK Biobank they call it 'Sex' too, I'll change it back!

Comment (sandrinebedard) [reply pipeline_ukbiobank/cli/compute_stats.py]: Done in https://github.com/sct-pipeline/ukbiobank-spinalcord-csa/pull/74/commits/654fd510ebc4e7d48ffbce6613b55ab8eb18ecbe

**Review by jcohenadad — APPROVED** [review]
awesome addition! and the scatterplot looks great 🚀

**Comment by sandrinebedard** [comment]
I also generated the scatterplot sperated for sex so we can see its effect on each parameter for fun 😊:
image

**Comment by jcohenadad** [comment]
> I also generated the scatterplot sperated for sex so we can see its effect on each parameter for fun 😊:
> image

wow, super interesting!!

**Pull request merged**